src/IROperator.h

```diff
@@ -188,6 +188,10 @@ struct BufferBuilder {
     EXPORT Expr build() const;
 };
 
+/** If e is a ramp expression with stride, default 1, return the base,
+ * otherwise undefined. */
+Expr strided_ramp_base(Expr e, int stride = 1);
```

Comment:
Even more confusing, I don't see that any call site uses anything except the default. @zvookin, was there an intended use for this that went missing, or should we just simplify it away?